packages/core/src/__integrationtests__/utils.ts

```diff
@@ -67,3 +90,111 @@ export const IsOfficialLicenseAuthority = CType.fromSchema({
   },
   type: 'object',
 })
+
+export async function initialTransfer(
+  faucet: KeyringPair,
+  addresses: Array<string>
+): Promise<void> {
+  await Promise.all(
+    addresses.map((address) =>
+      Balance.makeTransfer(address, ENDOWMENT).then((tx) =>
+        BlockchainUtils.signAndSubmitTx(tx, faucet, {
+          resolveOn: BlockchainUtils.IS_FINALIZED,
+          reSign: true,
+        }).catch((e) => console.log(e))
+      )
+    )
+  )
+}
+
+export async function getDidDeposit(didIdentifier: string): Promise<BN> {
+  const { api } = await BlockchainApiConnection.getConnectionOrConnect()
+  const result = await api.query.did.did<Option<any>>(didIdentifier)
+  DecoderUtils.assertCodecIsType(result, ['Option<DidDidDetails>'])
+  return result.isSome ? result.unwrap().deposit.amount.toBn() : new BN(0)
+}
+
+export async function getAttestationDeposit(claimHash: string): Promise<BN> {
+  const { api } = await BlockchainApiConnection.getConnectionOrConnect()
+  const result = await api.query.attestation.attestations<
+    Option<AttestationDetails>
+  >(claimHash)
+
+  DecoderUtils.assertCodecIsType(result, [
+    'Option<AttestationAttestationsAttestationDetails>',
+  ])
+  return result.isSome ? result.unwrap().deposit.amount.toBn() : new BN(0)
+}
+
+export async function createAttestation(
+  identity: KeyringPair,
+  requestForAttestation: IRequestForAttestation,
+  fullDid: FullDidDetails,
+  keystore: DemoKeystore
+): Promise<void> {
+  const attestation = Attestation.fromRequestAndDid(
+    requestForAttestation,
+    fullDid.did
+  )
+
+  const tx = await attestation.store()
+  const authorizedTx = await fullDid.authorizeExtrinsic(
+    tx,
+    keystore,
+    identity.address
+  )
+
+  await BlockchainUtils.signAndSubmitTx(authorizedTx, identity, {
+    resolveOn: BlockchainUtils.IS_FINALIZED,
+  })
+}
+
+export async function createMinimalFullDidFromLightDid(
```

Comment:
Same here. Are all these functions not accessible by adding the DID package as a dev dependency?

Reply:
Unfortunately, due to the number of times, it would be convenient to have this helper function.